response.go

```diff
@@ -586,6 +586,31 @@ func NewJsonResponse(status int, body any) (*http.Response, error) { // nolint:
 	return response, nil
 }
 
+// NewJsonResponseOrPanic is like [NewJsonResponse] but panics in case or error.
+//
+// It simplifies the call of [ResponderFromMultipleResponses], avoiding the use of a
+// temporary variable and an error check, and so can be used in such context:
+//
+//	httpmock.RegisterResponder(
+//	  "GET",
+//	  "/test/path",
+//	  httpmock.ResponderFromMultipleResponses([]*http.Response{
+//		httpmock.NewJsonResponseOrPanic(200, &MyFirstResponseBody),
+//		httpmock.NewJsonResponseOrPanic(200, &MySecondResponseBody),
+//	  }),
+//	)
+//
+// To pass the content of an existing file as body use [File] as in:
+//
+//	httpmock.NewJsonResponseOrPanic(200, httpmock.File("body.json"))
+func NewJsonResponseOrPanic(status int, body any) *http.Response { // nolint: revive
```

Comment:
New occurrences of nolint directive  should avoid space between `//` and `nolint`:
```suggestion
func NewJsonResponseOrPanic(status int, body any) *http.Response { //nolint: revive
```

Reply:
Shouldn't we apply this to the whole file for consistency ?

Reply:
I already did it 2 days ago.